compressed_image_transport/cfg/CompressedSubscriber.cfg

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python
+
+PACKAGE='compressed_image_transport'
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+mode_enum = gen.enum( [gen.const("unchanged", str_t, "unchanged", "keep image encoding"),
```

Comment:
if it's using strings, let's keep strings then :)
But in that case, in the C++ code, please use the created variables (```CompressedSubscriber_unchanged``` and so on, defined at the bottom of it normally). And yes, the rest of the C++ code does not seem to use ```CompressedPublisher_jpeg``` and ```CompressedPublisher_png``` .....